README.md

```diff
@@ -5,19 +5,69 @@
+a NetworkX representation of the connectome as a complex graph enables
+questions to be asked about nearest neighbors of a given neuron.
+An RDF semantic graph representation is useful for reading and
+writing annotations about multiple aspects of a neuron, such as what papers
+have been written about it, properties it may have such as
+ion channels and neurotransmitter receptors, etc.  A NeuroML representation is useful
+for answering questions about model morphology and simulation parameters.  A
+Blender representation is a full 3D shape definition that can be used for
+calculations in 3D space.
+
+The diversity of underlying representations required for OpenWorm
+presents a challenge for data integration and consolidation.  PyOpenWorm solves
+this challenge with a unified data access layer whereby different representations
+are encapsulated into an abstract view.  This allows the user to work with
+objects related to the *biological reality of the worm*, and
+forget about which representation is being used under the hood.  The worm
+itself has a unified sense of neurons, networks, muscles,
+ion channels, etc. and so should our code.
+
+Syntactical constructs in PyOpenWorm reflect the structure of
+the corresponding biological knowledge-base
+rather than the programming idioms of the underlying technologies. Wherever possible,
+pure Python libraries or those with few compilation requirements are used.
+
+Relationship to ChannelWorm
+-----------------------------
+[ChannelWorm](https://github.com/openworm/ChannelWorm) is
+the sub-project of OpenWorm which houses ion channel models.  In the future,
+we expect ChannelWorm to be a "consumer" of PyOpenWorm.  A PyOpenWorm database will house
+physical models, the digitized plots they are derived from (there is a Plot type in PyOpenWorm),
+and provide code to put those models into enumerated formats along with auxillary
+files or comments.  However, because these projects were not developed sequentially,
+there is currently some overlap in functionality, and PyOpenWorm itself houses
+a fairly substantial amount of physiological information about *C. elegans.*
+Ultimately, the pure core of PyOpenWorm, which is meant to be a data framework
+for storing metadata and provenance (i.e. parameters and trajectories
+associated with simulations), will be separated out into standalone functionality.
 
-What does it do?
-----------------
+Versioning data as code
+-----------------------
+
+A library that attempts to reliably expose dynamic data can often be broken because
+the underlying data sets that define it change over time.  This is because data
+changes can cause queries to return different answers than before, causing
+unpredictable behavior.
 
-Enables a simple Python API for asking various questions about the cells of the
-*C. elegans*, enabling the sharing of data about *C. elegans* for the purpose
-of building a [data-to-model pipeline](http://docs.openworm.org/en/latest/projects)
```

Comment:
I would keep reference to the pipeline. It is, in a nutshell, what PyOpenWorm is for. Also, the chart on that page locates PyOpenWorm within OpenWorm clearly.